packages/x-charts-pro/src/FunnelChart/funnelSlots.types.ts

```diff
@@ -10,14 +16,18 @@ export interface FunnelChartSlots
     ChartsLegendSlots,
     ChartsTooltipSlots,
     ChartsOverlaySlots,
-    ChartsAxisSlots {}
+    ChartsAxisSlots,
+    ChartsToolbarSlots,
+    Partial<ChartsSlots> {}
```

Comment:
Is it to anticipate the addition of chart export for the Funnel in an upcoming PR?

Because here you add slots that have no effect on the chart

Reply:
Which slots have no effect on the funnel? 

These are the toolbar's slots, can't all charts accept a toolbar, in theory?